central/alert/mappings/options.go

```diff
@@ -14,7 +14,7 @@ var OptionsMap search.OptionsMap
 
 func init() {
 	OptionsMap = search.Walk(v1.SearchCategory_ALERTS, "list_alert", (*storage.ListAlert)(nil))
-	alertOptions := schema.AlertsSchema.OptionsMap
+	alertOptions := schema.AlertsSchema.OptionsMap.Clone()
```

Comment:
I believe the intention here was just to update the `ListAlert's` OptionsMap to have same fields as `Alert's` OptionsMap as `Alert` is what we actually store in the db. I don't think the intention was to prevent the Alert OptionsMap getting updated in the process. So I added a clone here.

This way, when new fields are added to `Alert`, we would be able to use them in queries internally.